web/tests/test_discount.py

```
@@ -0,0 +1,93 @@
+            price=100.00,
+            allow_individual_sessions=False,
+            invite_only=False,
+            status="published",
+            max_students=10,
+            subject=self.subject,
+            level="beginner",
+            tags="",
+            is_featured=False,
+        )
+
+    def test_missing_course_id(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_missing_course_id`

Add return type annotation: `None`

(ANN201)